Evaluation/workflow.R

```diff
@@ -0,0 +1,13 @@
+# 
+# 1. Locate the template file qualification-workflow-template.R which is part of the installation of the R package ospsuite.reportingengine
+#    Once the package ospsuite.reportingengine is installed: type in R the command below to find out the location of the template
+#
+#    system.file("extdata", package = "ospsuite.reportingengine")
+#
+# 2. Replace the content of this file with the content of the template qualification-workflow-template.R and adjust it after that if required
+# 
+# 3. Create a qualification report in R via the commands below:
+#
+# source("workflow.R", encoding="UTF-8")
+# createQualificationReport(...)
+

```

Comment:
I don't want to duplicate the template, which might change with the future RE releases.
So here just a HowTo where to find the template and how to call it.